frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar/QuestionDetails.tsx

```diff
@@ -54,65 +67,83 @@ export const QuestionDetails = ({ question }: { question: Question }) => {
           </Flex>
         )}
       </SidesheetCardSection>
-      <SidesheetCardSection title={t`Saved in`}>
-        <Flex gap="sm" align="top" color="var(--mb-color-brand)">
-          <Icon
-            name="folder"
-            color="var(--mb-color-brand)"
-            className={SidebarStyles.IconMargin}
-          />
-          <Text>
-            <Link
-              to={`/collection/${question.collection()?.id}`}
-              variant="brand"
-            >
-              {question.collection()?.name}
-            </Link>
-          </Text>
-        </Flex>
-      </SidesheetCardSection>
+      {collection && (
+        <SidesheetCardSection
+          title={c(
+            "This is a heading that appears above the name of a collection - a collection that a dashboard is saved in. Feel free to translate this heading as though it said 'Saved in collection', if you think that would make more sense in your language.",
+          ).t`Saved in`}
```

Comment:
i hate what this does to the code. I wonder if we can think about ways to make context better